modules/custom/az_person/config/install/views.view.az_person.yml

```diff
@@ -107,25 +108,79 @@ display:
           items_per_page: 0
           total_pages: null
           id: 0
-          pagination_heading_level: h4
       exposed_form:
-        type: basic
+        type: az_better_exposed_filters
```

Comment:
Wondering if this is an expected change (?) 🤔

Reply:
I think this represents `az_finder` being enabled during export, maybe...

Reply:
Correct, this view should be exported without the finder module enabled.

Reply:
👍